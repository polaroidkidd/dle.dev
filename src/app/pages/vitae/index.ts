import { Vitae } from "./vitae";

export { Vitae };