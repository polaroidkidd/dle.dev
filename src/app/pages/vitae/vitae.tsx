import React from "react";

const Vitae: React.FC = () => {
  return (
    <div>VITAE</div>
  );
};
export { Vitae};