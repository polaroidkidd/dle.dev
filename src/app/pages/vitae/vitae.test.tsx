import React from "react";
import renderer from "react-test-renderer";
import { Vitae } from "./vitae";

test("Render Vitae Component", () => {
  const tree = renderer.create(
    <Vitae/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
