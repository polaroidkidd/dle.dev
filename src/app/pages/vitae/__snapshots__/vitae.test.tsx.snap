// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Vitae Component 1`] = `
<div>
  VITAE
</div>
`;
